cumulusci.yml

```
@@ -938,6 +938,40 @@ flows:
             2:
                 task: update_admin_profile
 
+    enable_gift_entry:
+        description: Enable the Gift Entry feature (unmanaged code only).
```

I think we can drop the part about this only working for unmanaged code in both of these flows.
```suggestion
        description: Enable the Gift Entry feature.
```